neuro_san/internals/graph/persistence/agent_filetree_mapper.py

```diff
@@ -0,0 +1,35 @@
+
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+from pathlib import Path
+
+from neuro_san.interfaces.agent_name_mapper import AgentNameMapper
+
```

Comment:
Simple default implementation of AgentNameMapper policy,
where agent name in the form abc/xyz/agent maps to its definition file location:
registry-root/abc/xyz/agent.hocon.